app/views/admin/invitations/index.html.erb

```diff
@@ -80,8 +81,13 @@
             <%= render "admin/users/index/invitation_actions_dropdown", user: user, context: "large" %>
           </td>
         </tr>
+      </tbody>
+      <% end %>
+      <% else %>
+      <td>
+        <%= render "admin/invitations/empty_state" %>
```

Comment:
So, I'm running into something interesting here... I am unable to place the "empty state" where I need it (centered within the table body, beneath the table headings, per the design). I am continually running into a situation where the "empty state" is placed above the table headings, centered and looking as expected, _or_ it is within the table body, but is not centered and does is not looking as expected--I've included screenshots below for reference:

Placed above the table headings, centered and looking as expected:
![Screen Shot 2022-05-04 at 1 44 22 PM](https://user-images.githubusercontent.com/32834804/166814682-979c0b08-a836-457c-8796-546a0ec2eebf.png)

Within the table body, but not centered or looking as expected:
![Screen Shot 2022-05-04 at 1 44 02 PM](https://user-images.githubusercontent.com/32834804/166814702-3b877f11-4d81-486a-956b-740da96ae046.png)

Any thoughts on how to get around this? It's clearly the table that is causing issues, as the "empty state" looks as expected when it renders outside of the table columns/rows, but the design shows the "empty state" within the table, beneath the table headers. 🤔